package.json

```diff
@@ -71,9 +71,11 @@
     "@mui/material": "^5.10.6",
     "@mui/system": "^5.10.6",
     "@types/react-dom": "^18.0.5",
+    "cron-validate": "^1.4.3",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
-    "react-select": "^5.4.0"
+    "react-select": "^5.4.0",
```

Comment:
Is react-select used anywhere? Only see Select being imported from mui in the code

Reply:
I previously used it for the environment selector; it shouldn't be needed anymore now that we use MUI instead.